Remove 'Sending discover' info message

In NetCOM, the DHCP client is started in the foreground, and the
respawnd mechanism puts it in the background. But the DHCP client
still thinks it runs in the foreground, so it keeps sending debug
messages on stdout. The 'Sending discover' message is particular
annoying, since it gets repeated every few seconds.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Index: busybox-1.18.4/networking/udhcp/dhcpc.c
===================================================================
--- busybox-1.18.4.orig/networking/udhcp/dhcpc.c
+++ busybox-1.18.4/networking/udhcp/dhcpc.c
@@ -453,7 +453,9 @@
 	 */
 	add_client_options(&packet);
 
+#if 0
 	bb_info_msg("Sending discover...");
+#endif
 	return raw_bcast_from_client_config_ifindex(&packet);
 }
 
